com.reprezen.swagedit.openapi3/src/com/reprezen/swagedit/openapi3/assist/OpenApi3ReferenceProposalProvider.java

```diff
@@ -29,7 +29,7 @@ public OpenApi3ReferenceProposalProvider() {
             // e.g. "/components/callbacks/myWebhook/$request.body#~1url/$ref"
             + "|/components/callbacks/" + COMPONENT_NAME_REGEX + "/[^/]+/\\$ref"; // in callbacks
     protected static final String PARAMETER_REGEX = ".*/parameters/\\d+/\\$ref";
-    protected static final String RESPONSE_REGEX = ".*/responses/\\d+/\\$ref";
+    protected static final String RESPONSE_REGEX = ".*/responses/((\\d{3}|\\dXX)|default)/\\$ref";
```

Comment:
This does no support response codes with a single wildcard, e.g. ` 20X`. 
What don't we use the regex from the schema: `"^([0-9X]{3})$"` (+ `|default`) ?